src/App.tsx

```diff
@@ -1,12 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { GoodsList } from './components/GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+
+  const pickGoods = (getGoods: () => Promise<Good[]>) => {
+    getGoods()
+      .then(recievedGoods => setGoods(recievedGoods));
```

Comment:
async await gives you a simpler way to work with asynchronous promise-based code